src/cascadia/TerminalApp/App.cpp

```diff
@@ -254,6 +273,10 @@ namespace winrt::TerminalApp::implementation
             settingsItem.Click({ this, &App::_SettingsButtonOnClick });
             newTabFlyout.Items().Append(settingsItem);
 
+            // Using alternate method here than VirtualKey due to https://github.com/microsoft/microsoft-ui-xaml/issues/708
+            // this should really use KeyboardAccelerator API bug above bug in framework prevents it, using override
+            settingsItem.KeyboardAcceleratorTextOverride(L"Ctrl + ,");

```

Comment:
This obviously complicates the lookup of the actual keybinding. We'd probably need to encapsulate the lookup and application of the accelerator into a method, that special-cases comma as a vkey.

Reply:
Sounds good...I'd like to resolve the above item first before seeing the best way to do this.